src/renderer/components/modals/BackupContents.tsx

```diff
@@ -81,15 +71,9 @@ export const BackupContents = (props: ModalContentsProps) => {
 				/>
 
 				<Title size="m" style={{ paddingTop: 15 }}>Ignore files</Title>
-				<p style={{ marginTop: 7 }}>Add any files(s) you would like to exclude from this backup.</p>
-
-				<TextButton
-					style={{ marginTop: 5 }}
-					className={styles.NoPaddingLeft}
-					onClick={() => onClickEditIgnore(site)}
-				>
-					Edit files to ignore
```

Comment:
[Here's](https://wpengine.atlassian.net/browse/LOC-5459?focusedCommentId=1007547) a comment about what this did.